task3/PhotoEnhancer/PhotoEnhancer/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Drawing;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace PhotoEnhancer
+{
+    static class Program
+    {
```

Comment:
Для правильной работы фильтра шума как я писал ранее в комментариях к заданию 2, нужно сделать ГСЧ (генератор случайных чисел) статическим полем класса, а не создавать такой объект заново для каждого пиксела.
Поскольку метод обработки пиксела переехал в метод `Main()` класса `Program`, то то его нужно переместить сюда
```suggestion
    {
        static Random rnd = new Random();
```